lib/x11test.pm

```diff
@@ -345,7 +346,9 @@ sub start_evolution {
     }
     send_key $self->{next};
     assert_screen "evolution_wizard-restore-backup";
-    send_key $self->{next};
+    wait_still_screen{
```

Comment:
maybe you misunderstanding the timing. `wait_still_screen` will ensure there is a still screen when the scope is left, i.e. *first* the key is pressed, *then* the screen is checked. Probably you expected the opposite. If the key press is lost then checking for a still screen afterwards will not make a difference.

Reply:
Would this be better?
```send_key_until_needlematch("evolution_wizard-identity",$self->{next},$counter);```

Reply:
that could work but I consider this a (messy) workaround for a problem that seems to be not fully understood. I suggest you look for related tickets first and see if you can help with the investigation of the real problem: https://progress.opensuse.org/projects/openqatests/issues?utf8=%E2%9C%93&set_filter=1&f%5B%5D=status_id&op%5Bstatus_id%5D=o&f%5B%5D=subject&op%5Bsubject%5D=%7E&v%5Bsubject%5D%5B%5D=wayland&f%5B%5D=&c%5B%5D=subject&c%5B%5D=project&c%5B%5D=status&c%5B%5D=assigned_to&c%5B%5D=fixed_version&c%5B%5D=is_private&c%5B%5D=due_date&c%5B%5D=relations&group_by=

the main ticket is probably https://progress.opensuse.org/issues/42362

Reply:
Would clicking the buttons, were available, to avoid the keyboard also be a messy workaround?

Reply:
probably not but harder to maintain as you would need to create needles for multiple different products and variants